stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageText.kt

```diff
@@ -64,17 +67,28 @@ public fun MessageText(
     val styledText = buildAnnotatedMessageText(message)
     val annotations = styledText.getStringAnnotations(0, styledText.lastIndex)
 
+    val isMyMessage = message.isMine()
+
+    val isEmojiOnly = message.isEmojiOnly()
+    val isSingleEmoji = message.isSingleEmoji()
+
+    val style = when {
+        isSingleEmoji -> ChatTheme.typography.singleEmoji
+        isEmojiOnly -> ChatTheme.typography.emojiOnly
+        else -> ChatTheme.typography.bodyBold
+    }
+
     if (annotations.isNotEmpty()) {
         ClickableText(
             modifier = modifier
                 .padding(
-                    start = 12.dp,
-                    end = 12.dp,
+                    start = if (isEmojiOnly && !isMyMessage) 0.dp else 12.dp,
```

Comment:
So I think this case can't happen - annotations won't be empty only in case we have an annotation in the text, which currently is only supporting links. 

This might be possible if include markdown, but even then we could add different types of tags for links to distinguish them.

So I think we can leave this case out of the emoji padding stuff.